editor/src/clj/editor/pipeline/fontc.clj

```diff
@@ -597,8 +603,8 @@
             (aset image out-shadow-offset (unchecked-byte df-outline-channel)))))
       (when (and (> shadow-alpha 0.0) (> shadow-blur 0) (> channel-count 1))
         (let [image-byte-length (* width height channel-count)
-              shadow-image' (byte-array image-byte-length)
-              shadow-image (do (System/arraycopy image 0 shadow-image' 0 image-byte-length) shadow-image')
+              shadow-image (byte-array image-byte-length)
+              _ (System/arraycopy image 0 shadow-image 0 image-byte-length)
```

Comment:
Unwrapped the `(do ... shadow-image')` expression here, since Cursive couldn't infer out the correct `DataBufferByte` constructor from the types on the next line. Instead, I do the `arrayCopy` call inline and ignore its return value.